data/preset/course.yaml

```diff
@@ -0,0 +1,150 @@
+---
+defines:
+
+- &main-arrangement-slot
+  arrangement:
+    type: Arrangement.Grid
+    properties:
+      homogeneous: true
+      max-children-per-line: 3
+      min-children-per-line: 3
+    slots:
+      card:
```

Comment:
general comment: on "leaves" of the tree you can leave out `shortdef` for compactness, like `card: 'Card.Thumb(background-size: "center")'` but that's a matter of taste so I will certainly not gripe about that